### ISSUE TITLE
Class extending

### ISSUE THREAD

**Issue by gareth-ib** [dan-da/py2php#8]
```py
class BinFactory(object):
```
converts to
```php
class BinFactory extends object {
```
which definitely is some sort of issue, `object` would have to be an existing class. I think it has to do with class overriding or something? Not sure, I don't code python :)

**Comment by gareth-ib** [comment]
nevermind, I see what's going on.. maybe omit the system objects? I dunno

**Comment by dan-da** [comment]
Fixed in b5874b8b1d919a0957fbd9cbc7d6d1a4518302af.  Now it translates `object` to `stdClass` when inheriting.
